src/modules/sync/types.ts

```diff
@@ -1,3 +1,10 @@
 import { GRANULAR_SYNC_FOLDERS } from 'src/modules/sync/constants';
 
 export type GranularSyncFolders = ( typeof GRANULAR_SYNC_FOLDERS )[ number ];
+
+export type RawFileNode = {
```

Comment:
```suggestion
export type RawDirectoryEntry = {
```
It doesn't have to be this exact name, but I think `RawFileNode` is a little misleading when it can also represent a directory. The name I suggested is inspired by node's `fs.Dirent` type (see [docs](https://nodejs.org/api/fs.html#class-fsdirent))

Reply:
https://github.com/Automattic/studio/pull/1940/commits/c42857775a5d6882f8aa24ee5b9f2866671073d9